CMakeLists.txt

```diff
@@ -189,15 +189,16 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "QNX")
   SET(EXTENDED_MEDIA_MODE OFF)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
   add_definitions(-DOS_WIN32)
+  add_definitions(-DOS_WINDOWS)
   add_definitions(-DWIN32)
   add_definitions(-D_CRT_SECURE_NO_WARNINGS)
   add_definitions(-DWIN32_LEAN_AND_MEAN)
   add_definitions(-DASR_HAS_VR)
   SET(BUILD_AVAHI_SUPPORT OFF)
-  SET(BUILD_USB_SUPPORT OFF)
+  SET(BUILD_USB_SUPPORT ON)
   SET(BUILD_BT_SUPPORT OFF)
```

Comment:
Please use lowercase command (Change from SET to set)in CMake to keep use same command format.